src/index.html

```diff
@@ -2,11 +2,428 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
+    <title>Moyo header</title>
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="index.html">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__bar">
+          <li class="nav__list">
+            <a class="nav__links is-active" href="/">apple</a>
+          </li>
+
+          <li class="nav__list">
+            <a class="nav__links" href="/">samsung</a>
+          </li>
+
+          <li class="nav__list">
+            <a class="nav__links" href="/">smartphones</a>
+          </li>
+
+          <li class="nav__list">
+            <a
+            class="nav__links"
+            data-qa="nav-hover"
+            href="/"
+            >
+            laptops & computers
+            </a>
```

Comment:
don't forget about code style
fix in all cases
```suggestion
            <a
              class="nav__links"
              data-qa="nav-hover"
              href="/"
            >
              laptops & computers
            </a>
```